services/sidecar/src/simcore_service_sidecar/executor.py

```diff
@@ -293,77 +290,94 @@ async def _run_container(self):
-                config=docker_container_config
-            )
-            # start monitoring logs
-            log_file = self.shared_folders.log_folder / "log.dat"
-            if self.integration_version == version.parse("0.0.0"):
-                # touch output file, so it's ready for the container (v0)
-                log_file.touch()
-
-                log_processor_task = fire_and_forget_task(
-                    monitor_logs_task(log_file, self._post_messages)
+            async with aiodocker.Docker() as docker_client:
+                await self._post_messages(
+                    LogType.LOG,
+                    f"[sidecar]Running {self.task.image['name']}:{self.task.image['tag']}...",
                 )
-            else:
-                log_processor_task = fire_and_forget_task(
-                    monitor_logs_task(container, self._post_messages, log_file)
+                container = await docker_client.containers.create(
+                    config=docker_container_config
+                )
+                # start monitoring logs
+                await self._start_monitoring(container)
+
+                # start the container
+                await container.start()
+                # indicate container is started
+                await self.rabbit_mq.post_instrumentation_message(
+                    {
+                        "metrics": "service_started",
+                        "user_id": self.user_id,
+                        "project_id": self.task.project_id,
+                        "service_uuid": self.task.node_id,
+                        "service_type": "COMPUTATIONAL",
+                        "service_key": self.task.image["name"],
+                        "service_tag": self.task.image["tag"],
+                    }
                 )
-            # start the container
-            await container.start()
-            # indicate container is started
-            await self.rabbit_mq.post_instrumentation_message(
-                {
-                    "metrics": "service_started",
-                    "user_id": self.user_id,
-                    "project_id": self.task.project_id,
-                    "service_uuid": self.task.node_id,
-                    "service_type": "COMPUTATIONAL",
-                    "service_key": self.task.image["name"],
-                    "service_tag": self.task.image["tag"],
-                }
-            )
 
-            # wait until the container finished, either success or fail or timeout
-            container_data = await container.show()
-            while container_data["State"]["Running"]:
-                await asyncio.sleep(2)
-                # reload container data
+                # wait until the container finished, either success or fail or timeout
+                container_data = await container.show()
+                while container_data["State"]["Running"]:
+                    await asyncio.sleep(2)
```

Comment:
MINOR: named constant that documents why to sleep two secs here ?? e.g.
```python
  TIME_TO_NEXT_PERIODIC_CHECK_SECS = 2
  while container_data["State"]["Running"]:
       awaut asyncio.sleep( TIME_TO_NEXT_PERIODIC_CHECK_SECS )
```

Reply:
done. you astonish me with such long names :rofl: